src/pages/deployer/data.ts

```diff
@@ -41,15 +41,15 @@ const onchainFormSpec = [
   {
     name: "description",
     label: "Description",
-    description: "Optional freeform sentence explaining about your project",
+    description: "Optional sentence explaining about your project.",
     type: "string",
     default: "Low fee peer-to-peer electronic cash alternative to Bitcoin",
   },
 
   {
     name: "tokenImage",
-    label: "Jetton Logo URI",
-    description: "URL of 256x256 pixel PNG image of token logo with transparent background",
```

Comment:
why did we remove "with transparent background"?

Reply:
The text was modified according to the text in Figma template

Reply:
similar - let's restore pls